intersection_transit_maneuvering/include/call_test.h

```diff
@@ -0,0 +1,76 @@
+#pragma once
+/*
+ * Copyright (C) 2021 LEIDOS.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+#include <ros/ros.h>
+#include <vector>
+#include <cav_msgs/TrajectoryPlan.h>
+#include <cav_msgs/TrajectoryPlanPoint.h>
+#include <cav_msgs/Maneuver.h>
+#include <cav_msgs/Plugin.h>
+#include <boost/shared_ptr.hpp>
+#include <carma_utils/CARMAUtils.h>
+#include <boost/geometry.hpp>
+#include <carma_wm/Geometry.h>
+#include <cav_srvs/PlanTrajectory.h>
+#include <itm_helper.h>
+
+
+namespace call_test
+{
+class CallTest: public Interface
+{
+    public:
+        /**
+         * @brief Construct a call() function to use for unit testing
+         * 
+         */
+        CallTest();
+
+        /**
+         * @brief Test call() function for unit testing
+         * 
+         * @param req Incoming PlanTrajectory service request 
+         * @param resp Incoming PlanTrajectory service response
+         * @return true 
+         * @return false 
```

Comment:
For @return comments you would normal describe the return behavior not list the possible values. 
So something like 
@return true if method completes successfully, false otherwise